src/ui/pages/SwapForm/fieldsets/ReceiveTokenField/MarketAssetSelect/MarketAssetSelect.tsx

```diff
@@ -42,16 +44,23 @@ export function MarketAssetSelect({
       ),
     [addressPositions]
   );
+
+  const { data: popularAssets, isLoading: popularAssetsAreLoading } = useQuery({
+    queryKey: ['getPopularTokens', chain],
+    queryFn: () => getPopularTokens(chain),
+    suspense: false,
+    retry: false,
+    staleTime: ONE_DAY,
```

Comment:
does this make sense if this query isn't persisted?

Reply:
If it is not persisted, then it will be updated once every session which is ok, I think